visidata/loaders/xlsx.py

```diff
@@ -131,14 +131,18 @@ def save_xlsx(vd, p, *sheets):
             vd.warning(f'saving {vs.name} as {vs.xls_name}')
         ws = wb.create_sheet(title=vs.xls_name)
 
+        null_value = vs.options.get("null_value", vd.options.get("null_value", ""))
```

Comment:
You should just be able to use `vs.options.null_value`, which will fall back appropriately to the class value, global, etc.  Did it not work for you for some case?

Reply:
I didn't try that — though how do I then default to the empty string, if none of the values is set?

Reply:
Ah I see, the default value of `null_value` is None.  `null_value` is actually for which values should be counted as null during aggregations, and overloading it this way would probably be confusing.  I'm thinking that we can just use `""` specifically in the xls saver for the None case, and if we want to make it configurable, it would be via a new `xlsx_save_null` option.  Though I'm not sure that's necessary at the moment.

It does leave open the question, if `null_value` is set, should that value should be given to the savers as None when it appears?

Reply:
Right I'm new to visidata and there's only so much complexity one can take in at once — so thanks for your prompt feedback on this PR. I think, as result, there is a good path forward.

Given your comments, it seems visidata is not explicit in how it serializes `null` in formats that require strings. A format-agnostic `save_null` option seems reasonable for this — `None` values can be converted to the text of `save_null` when serialised.

The reverse is trickier: it is probably right not to convert values to `None` all the time, but it can occasionally be useful. This could be a separate feature request.

For this PR, how about just doing what you propose, and setting `None` values to the empty string?